app/frontend/src/router/index.js

```diff
@@ -38,6 +38,33 @@ export default function getRouter(basePath = '/') {
           hasLogin: true
         }
       },
+      {
+        path: '/admin',
+        component: () => import(/* webpackChunkName: "admin" */ '@/views/Admin.vue'),
+        children: [
+          {
+            path: '',
+            name: 'Admin',
+            component: () => import(/* webpackChunkName: "adminroot" */ '@/views/admin/Root.vue')
+          },
```

Comment:
Just thinking about how this page is behaving atm - on the root admin page, we have a tab system where you can look at 3 subpanels - I'm half considering whether we should have those nested subpanels be explicit routes too (when you visit one of the later admin pages, then hit back, it always goes back to first tab instead of remembering what state it was previously). This could be dealt with via route state, or we could shove the tab state into vuex under admin tree or something. Let's talk about this some more first.

Reply:
Changing tab to be remembered in state in next PR as discussed.